lib/px/service/client/circuit_breaker.rb

```diff
@@ -7,6 +7,10 @@ module CircuitBreaker
     included do
       include ::CircuitBreaker
 
+      cattr_accessor :handler do
```

Comment:
Lets leave the existing CircuitBreaker service object aside (and plan to remove it) and just build this into the client itself.  There is no time where we wouldn't want a circuit breaker, so it's simpler to just build it in.